tomviz/MainWindow.cxx

```diff
@@ -199,37 +199,32 @@ MainWindow::MainWindow(QWidget* _parent, Qt::WindowFlags _flags)
 
   // Build Tomography menu
   // ################################################################
-
   QAction *toggleDataTypeAction = ui.menuTomography->addAction("Toggle Data Type");
   ui.menuTomography->addSeparator();
 
   QAction *setTiltAnglesAction = ui.menuTomography->addAction("Set Tilt Angles");
   ui.menuTomography->addSeparator();
-    
-  QAction *generateTiltSeriesAction = ui.menuTomography->addAction("Generate Tilt Series");
-  ui.menuTomography->addSeparator();
 
   QAction *subtractBackgroundAction = ui.menuTomography->addAction("Background Subtraction (Manual)");
-  QAction *alignAction = ui.menuTomography->addAction("Translation Align");
-  QAction *autoAlignAction = ui.menuTomography->addAction("Translation Align (Auto)");
-  QAction *rotateAlignAction = ui.menuTomography->addAction("Rotation Align");
+  QAction *alignAction = ui.menuTomography->addAction("Tilt Series Registration (Manual)");
```

Comment:
Alignment is the typical word used instead of registration.

On Mon, Nov 23, 2015, 1:13 PM Marcus D. Hanwell <notifications@github.com>
wrote:

> In tomviz/MainWindow.cxx
> <https://github.com/OpenChemistry/tomviz/pull/269#discussion_r45638163>:
>
> >
> >    QAction *subtractBackgroundAction = ui.menuTomography->addAction("Background Subtraction (Manual)");
> > -  QAction *alignAction = ui.menuTomography->addAction("Translation Align");
> > -  QAction *autoAlignAction = ui.menuTomography->addAction("Translation Align (Auto)");
> > -  QAction *rotateAlignAction = ui.menuTomography->addAction("Rotation Align");
> > +  QAction *alignAction = ui.menuTomography->addAction("Tilt Series Registration (Manual)");
>
> Yes, I think image and rotation axis alignment would make more sense, and
> if it isn't standard for you then I think keeping the word alignment is far
> more likely to guide people to the thing they want than registration is.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenChemistry/tomviz/pull/269/files#r45638163>.
>